scripts/checkVomsTar.sh

```diff
@@ -29,6 +30,14 @@ if [ -e ${CMSSW_VERSION}.tar.gz ]; then
 	ls -lth ${CMSSW_VERSION}.tar.gz
 fi
 
-if [ -n "$XRDIR" ]; then
-	xrdcp -f ${CMSSW_VERSION}.tar.gz ${XRDIR}/${CMSSW_VERSION}.tar.gz
+if [[ "${XRDIR}" == *"root://"* ]]; then
+	CMD="xrdcp"
+elif [[ "${XRDIR}" == *"gsiftp://"* ]]; then
+	CMD="env -i X509_USER_PROXY=${X509_USER_PROXY} gfal-copy"
+else
+	echo "ERROR Unknown transfer protocol for the tarball"
```

Comment:
this should probably be followed by `exit 1`

Reply:
actually, won't this error get printed if XRDIR is empty? that's not desirable.

Instead, below where you check XRDIR and CMD, add another case where XRDIR is filled but CMD isn't, and then print the error message and `exit 1`

Reply:
What about the way it is now?

Reply:
That also seems fine; have you tested the different cases?

Reply:
Yes, I have tested the gfal, xrootd, and fallback cases. All three of them worked as expected.